components/optolink.rst

```diff
@@ -188,6 +199,13 @@ Configuration variables:
 - **map** (**Required**, map): Mapping of numerical value to human readable value.
 - All other options from :doc:`/components/switch/index`
 
+.. warning::
+
+    The optical interface has a limited throughput, ususally getting a response from the Viessmann controller takes 2-3 seconds and is sequential. Keep that in mind when defining sensors / control entities and define appropriate update interval (e.g. for 20 entities maximum frequency would be to update every 60s, for more entities the update interval should be longer). Example config:
+
```

Comment:
From a technical point of view this description is not totally correct. 
Your experience of 2-3 seconds result from the fact that a read request (and also a a write request) is placed in a queue which has a maximum length twice as long as the number of used datapoints. The request throughput is much faster than one request per 2-3 seconds. But in the case where there are many requests waiting in this queue the time from the end of the queue to the top of the queue (plus the processing time in the Viessmann controller) can definitely result in 2-3 seconds. 
Seen this way your calculation "20 entitites * 3 seconds result in an update_frequency of 60 seconds" is not correct.
What is totaly correct and definitely worth to be mentioned is "use appropriate update interval"!

Reply:
> From a technical point of view this description is not totally correct.

Well, maybe that is not fully correct, but even with long update intervals and small number of sensors I've never got response faster than the 2-3 seconds, hence my conclusion on recommended refresh interval for said 20 sensors. Anything more packed results in lost data or failed controls.